Tasks/NpmAuthenticate/task.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "ad884ca2-732e-4b85-b2d3-ed71bcbd2788",
+    "name": "npmAuthenticate",
+    "friendlyName": "npm Authenticate",
+    "description": "Authenticate a .npmrc file for tasks which use npm.",
+    "helpMarkDown": "",
+    "category": "Package",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "minimumAgentVersion": "2.115.0",    
+    "instanceNameFormat": "npm Authenticate",
+    "inputs": [
+        {
+            "name": "workingFile",
+            "label": ".npmrc file to authenticate",
+            "helpMarkDown": "Path to the target .npmrc file. Select the file, not the folder e.g. \"/packages/mypackage.npmrc\".",
```

Comment:
Path to the .npmrc file that specifies the registries you want to work with. Select the file, not the folder e.g. \"/packages/mypackage.npmrc\".